server.js

```diff
@@ -46,7 +46,7 @@ function create (app) {
 
 require('./lib/bootevent')(env).boot(function booted (ctx) {
     var app = require('./app')(env, ctx);
-    var server = create(app).listen(PORT);
+    var server = create(app).listen(PORT, '::');
```

Comment:
did you mean to include this?  I'm not sure what the behaviour will be in all cases environments